### ISSUE TITLE
Implement ProjectMembersView

### ISSUE THREAD

**Issue by jkibele** [NCEAS/metacatui#825]
This is a subtask of NCEAS/project-papers#35. ProjectMembersView will extend ProjectSectionView to produce something like this: 

![image](https://user-images.githubusercontent.com/4845845/47944712-64ba1480-deba-11e8-8659-4f63ceb4a2db.png)

**Comment by laurenwalker** [comment]
@jkibele --

In the metadata editor, we use the [EMLPartyView](https://github.com/NCEAS/metacatui/blob/master/src/js/views/metadata/EMLPartyView.js) to display a single EMLParty as part of the editor. I think a lot of this code can be reused for rendering people in the project views since the XML schema for our project XML documents imports the EMLParty schema (i.e. a party in an EML document will be written exactly the same way as a party in a project document).

I would even suggest we create a view called EMLPartiesView inside the /views/metadata/eml directory and make it project-agnostic so that it can be reused for EML display in the future, as well.

**Comment by jkibele** [comment]
Er, I thought this was ready for review, then realized the acknowledgements and awards stuff needs to go in. I'll give that a crack before marking it for review.

**Comment by jkibele** [comment]
Regarding the acknowledgements text, I noticed that there were two arrays in the model: `text` and `originalText`. I wasn't quite sure what the difference was so I used `text`. They seem to hold the same content in the example document.
 
I also assumed that it was an array so that it could store multiple paragraphs. So I put an `each` in the template to put each paragraph in a `<p>` tag.

The mock-up showed an "Awards" header in the same box with "Acknowledgements", but I didn't see any awards info in the SASAP example document. Did I miss something?

I also wasn't sure how to deal with the logos, so I've left that out for now.

**Comment by laurenwalker** [comment]
`text` is the right attribute to use for displaying the text from the model. `originalText` is used to store the original text content taken from the document. It's only really helpful in the editor where the text can be changed after the original document is parsed.

@robyngit or @jeanetteclark -- could you add an awards element to the SASAP document at some point?

@jkibele , you could also just upload your own example documents to test various parts of the members view that need testing. I think you mentioned earlier you needed a phone number or email to test as well? You could just take the SASAP document and add gibberish/[lorem ipsum](https://www.lipsum.com/) for any missing fields that need testing.

**Comment by robyngit** [comment]
I'll add an awards element to the SASAP document now.

@jkibele, if it helps, also feel free to add testing elements to the `markdownTest` xml doc.

**Comment by robyngit** [comment]
Award info is now included in the SASAP xml doc (https://github.com/NCEAS/project-papers/commit/616e05e1c948d091c2a115ae34e80d793f0693ee) which is updated on knb.dev, but the project model will need to be updated, see https://github.com/NCEAS/metacatui/issues/853.

**Comment by laurenwalker** [comment]
- [ ] Show award logos

- [ ] Add fax and phone numbers to example project to test rendering

**Comment by laurenwalker** [comment]
The following fields need to be tested in the display:
- phone number
- fax number
- address
- user id (e.g. an ORCID)
- online URL
- salutation

I added all of these fields to the example project, named `example`. (Navigate to `your-local-MetacatUI/projects/example`.)

**Comment by laurenwalker** [comment]
We could skip the fax and address for now to save time. Those can always be added later.

Generally, I would think if someone includes contact info in the project document, they find it important enough to display. But we can add it later.

**Comment by jkibele** [comment]
I added in `onlineUrl`s from the associatedParties. After allowing for multiple urls per project member, I found that the `EMLParty` model didn't actually parse multiple urls. 

If I understand [the schema](https://github.com/NCEAS/eml/blob/RELEASE_EML_2_1_1/eml-party.png) correctly, it should. I made a separate commit (0ca2c58741476163d73a672b2108f6588e0d2608) where I changed the parsing to allow for more than 1. However, I'm not confident that this change won't break something elsewhere. I'm not entirely sure where EMLParty model comes into play.

At any rate, my [previous commit](https://github.com/NCEAS/metacatui/commit/10f91513599f45f2e5f6977d1cbcf6ab73199c48) will work with or without the second one so if it breaks something, we can roll it back.
